udf/deanonymize_event.sql

```diff
@@ -0,0 +1,43 @@
+/*
+
+Rename struct fields in anonymous event tuples to meaningful names
+
+*/
+CREATE TEMP FUNCTION
+  udf_deanonymize_event(tuple STRUCT<
+    f0_ INT64,
+    f1_ STRING,
+    f2_ STRING,
+    f3_ STRING,
+    f4_ STRING,
+    f5_ ARRAY<STRUCT<key STRING, value STRING>>>) AS (
+    (SELECT
+      AS STRUCT tuple.f0_ AS `timestamp`,
```

Comment:
Elsewhere, we have prefixed all of these field names with `event_` which has some benefits in avoiding reserved words like `timestamp` and making the purpose of the field more obvious when unnesting. Should we do so here?

Also, we should add calls to this function into the views for `main` and other pings, but that can be a separate PR.

Reply:
I left these as unprefixed since these fields are still encapsulated within a struct, and it seemed a bit redundant in that context. I can see the argument for the adding the prefix, though for the flattened events dataset my plan was to manually add the prefix in the new query

Reply:
Although tbf the javascript udf _that I wrote a few weeks ago_ that parsed out events had the prefix and that output was also a struct, so I guess I have to go discuss this with my past self